### PR TITLE
roslynpad: Add version 15.1

### PR DESCRIPTION
**Pull request by gregmac**
> A cross-platform C# editor based on Roslyn and AvalonEdit

- https://roslynpad.net/ 
- https://github.com/aelij/RoslynPad

### CODE REVIEW

**Comment by gregmac** [comment]
@Ash258 You tagged this with `package-fix-needed` -- Is a change needed here, and if so, what?

File: bucket/roslynpad.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "15.1",
+    "description": "A cross-platform C# editor based on Roslyn and AvalonEdit",
+    "homepage": "https://roslynpad.net/",
+    "license": "Apache-2.0",
+    "url": "https://github.com/aelij/RoslynPad/releases/download/15.1/RoslynPad.zip",
```
Comment (Ash258) [inline bucket/roslynpad.json:6]:
```suggestion
    "suggest": {
        "dotnetRuntime": "windowsdesktop-runtime"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/aelij/RoslynPad/releases/download/15.1/RoslynPad.zip",
            "hash": "9d2f38a97bfc34b2918439b6265edec5460d9f86346b15d158d1c0f7d9cd3645"
        }
    },
```

File: bucket/roslynpad.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "version": "15.1",
+    "description": "A cross-platform C# editor based on Roslyn and AvalonEdit",
+    "homepage": "https://roslynpad.net/",
```
Comment (Ash258) [inline bucket/roslynpad.json:4]:
```suggestion
    "homepage": "https://roslynpad.net",
```

File: bucket/roslynpad.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        [
+            "RoslynPad.exe",
+            "RoslynPad"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/aelij/RoslynPad"
+    },
+    "autoupdate": {
+        "url": "https://github.com/aelij/RoslynPad/releases/download/$version/RoslynPad.zip"
```
Comment (Ash258) [inline bucket/roslynpad.json:18]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/aelij/RoslynPad/releases/download/$version/RoslynPad.zip"
            }
        }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Why all of of you are so impatient and cannot wait a few hours? Do you really think everyone has time to do review immediately you open PR??

A

**Comment by Ash258** [comment]
What does this mean? The very first thing on application startup

A

**Comment by issaclin32** [comment]
Works fine for me.
未命名 - 16

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**